Automattic/jetpack
# My Jetpack: Protect card- Reduce the count font-size as the number of digits increases.

This was a request by @ilonagl on the Design team:

> Question, For the "Logins blocked", is there any way to minimize the font when it gets to a big number? If not, then maybe we can reduce the size for the numbers? (See screenshot below)

![Screen Shot 2024-08-19 at 06 58 53](https://github.com/user-attachments/assets/d4e49165-9ee6-4237-a396-7193893d67ae)

This PR dynamically reduces the font-size of the "Logins blocked" numbers (and slightly reduces letter-spacing) as the number of digits increases.






Fixes https://github.com/Automattic/jetpack-marketing/issues/936

## Proposed changes:

* In the number of logins blocked, reduce the font size as the number of digits increases.
* Slightly reduce letter-spacing when logins blocked is more than 3 digits (> 999).
* Also added `numberFormat()` to the number of logins blocked, (Which adds commas to the number (for US locale)).

### Other information:


## Jetpack product discussion



The number of logins blocked in the My Jetpack Protect card was initially added as part of: --> PT: Add value to the Protect product card in My Jetpack: pbNhbs-aP6-p2


## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Checkout this branch on the Jetpack plugin via the Jetpack Beta plugin or on your local dev environment.
* Make sure Jetpack is connected (at least "site" connected).
* If using a Jurassic Ninja site, SSH into the site using the site credentials displayed at the top of the WordPress dashboard. If using local dev environment, just open your terminal and prepend `jetpack docker` to the WP CLI command stated in the next step.
* Run the WP CLI command: `wp option update jetpack_protect_blocked_attempts 12345`
* Open the My Jetpack page and view the Protect product card.  Blocked logins should be formatted to say "12.3K" and the font-size should be 32px. (See screenshot:)
![Screen Shot 2024-08-21 at 07 22 26](https://github.com/user-attachments/assets/7589248b-434b-4bdc-827e-0ee433969938)
